extensions-contrib/moving-average-query/src/test/java/org/apache/druid/query/movingaverage/averagers/DoubleSumAveragerFactoryTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.movingaverage.averagers;
+
+import org.hamcrest.core.IsInstanceOf;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DoubleSumAveragerFactoryTest
+{
+
+  @Test
+  public void testCreateAverager()
+  {
+    AveragerFactory<?, ?> fac = new DoubleSumAveragerFactory("test", 5, 1, "field");
+    Assert.assertThat(fac.createAverager(), IsInstanceOf.instanceOf(DoubleSumAverager.class));
+  }
+
```

Comment:
It's missing from the other `movingaverager` factory tests, but a nice to have for the new ones being added is a test for the JSON serialization/deserialization, similar to `org.apache.druid.firehose.azure.StaticAzureBlobStoreFirehoseFactoryTest#testSerde()`. To make the test work, you'll also need to add `equals` and `hashCode` methods to the factory.

Reply:
@yurmix Are you planning to add serialization/deserialization tests? Otherwise, the PR LGTM.

Reply:
@ccaominh thank you. I tried to get to it but didn't find the time, I'll keep this in mind for the next more major change.